.travis.yml

```diff
@@ -143,7 +143,7 @@ install:
       elif [[ "$DISABLE_XSIMD" == 1 ]]; then
         cmake -DBUILD_TESTS=ON ..;
       elif [[ "$ENABLE_TBB" == 1 ]]; then
-        cmake -DDOWNLOAD_GTEST=ON -DXTENSOR_USE_XSIMD=ON -DXTENSOR_USE_TBB=ON -DTBB_INCLUDE_DIR=/home/travis/miniconda/include -DTBB_LIBRARY=/home/travis/miniconda/lib ..;
+        cmake -DXTENSOR_USE_XSIMD=ON -DBUILD_TESTS=ON -DXTENSOR_USE_TBB=ON -DTBB_INCLUDE_DIR=/home/travis/miniconda/include -DTBB_LIBRARY=/home/travis/miniconda/lib ..;
```

Comment:
Why removing the `DOWNLOAD_GTEST`?

Reply:
Because the gtest 1.8.1 does not build on recent compilers because of the new default `Werror=sign-conversion`, so I'm moving this to gcc 4.9.